app/views/components/common_atar.scala.html

```diff
@@ -0,0 +1,43 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.request.AuthenticatedRequest
+@import config.AppConfig
+
+@()(implicit request: AuthenticatedRequest[_], messages: Messages, appConfig: AppConfig)
+
+ <table id="atars">
```

Comment:
The table in atars, liabilities, correspondence, and misc are all the same.
The only difference is the cases that we will pass through, and the ID's I suppose.

Could we make it a single component where we pass in an id-prefix, and later the cases?

Reply:
It seems from the prototype that the tables have different headers, hence the need for different components.

Reply:
Ah my bad, you are correct!